### ISSUE TITLE
Improvement on the email configure page

### ISSUE THREAD

**Issue by BirkhoffLee** [NamelessMC/Nameless#597]
* What kind of issue is it (Question, Bug, etc.)?
Improvement, or future-request

* In which version of NamelessMC did it occur (1.x.x)?
1.0.15

* How would you describe the issue? 
On the email sender configuration page (`/admin/core/?view=email`), the password field will be intended left blank -- the password is still in the configuration file, it just don't show up. This is absolutely good, but when I change anything else on the page and submit the changes, the password in the configuration file just change to blank because the field was left blank by the system intendedly.

In conclusion: Please make something that if the password field is left blank, don't actually change the password in the configuration file.

**Comment by samerton** [comment]
Thanks for spotting this
